src/modules/previewpane/SvgThumbnailProvider/SvgThumbnailProvider.cs

```diff
@@ -166,6 +166,8 @@ public static string WrapSVGInHTML(string svg)
                     {0}
                   </body>
                 </html>";
+
+            // Using CurrentCulture since this is user facing
```

Comment:
I'm not entirely sure about this one, since it should in theory just be copying the .svg file's content verbatim. @ryanbodrug-microsoft and @somil55 might have more context on this.

Reply:
@arjunbalgovind you are correct. This is SVG file's content and should be displayed as it is.

Reply:
Does that mean the cultureinfo and comment are correct here? just to confirm

Reply:
No, I think it should be Ordinal/Invariant